include/aspect/postprocess/geoid.h

```diff
@@ -118,9 +123,24 @@ namespace aspect
         double density_below;
 
         /**
-         * A parameter to control whether to include the dynamic topography contribution on geoid
+         * A parameter to control whether to include the surface topography contribution on geoid
+         */
+        bool include_surface_topo_contribution;
+
+        /**
+         * A parameter to control whether to include the CMB topography contribution on geoid
+         */
+        bool include_CMB_topo_contribution;
+
+        /**
+         * A parameter to specify if the top boundary is an active free surface
+         */
+        bool use_freesurface_surface_topography;
```

Comment:
Please use @jaustermann 's suggested name instead:
```suggestion
        bool use_free_surface_topography;
```

Reply:
Sure? I think this naming 'use_free_surface_topography' is not so intuitive. Because it looks like ' use free surface' but that's only for the surface